app/commands.py

```diff
@@ -826,8 +826,8 @@ async def unsilence(ctx: Context) -> Optional[str]:
     if target.priv & Privileges.STAFF and not ctx.player.priv & Privileges.DEVELOPER:
         return "Only developers can manage staff members."
 
-    await target.unsilence(ctx.player)
-    return f"{target} was unsilenced."
+    await t.unsilence(ctx.player, " ".join(ctx.args[1:]))
```

Comment:
huh?

Reply:
oh I fucked up the merge conflict 1 sec

Reply:
we should at a minimum
```suggestion
    await t.unsilence(ctx.player, reason=" ".join(ctx.args[1:]))
```

Reply:
why that

Reply:
unrestrict doesn't have it either

Reply:
speaking of unrestrict, i am not a fan of having the join inline. would rather you put in in a variable

Reply:
fixed